packages/node/src/login/oidc/oidcHandlers/RefreshTokenOidcHandler.ts

```diff
@@ -222,6 +223,11 @@ export default class RefreshTokenOidcHandler implements IOidcHandler {
 
     return Object.assign(sessionInfo, {
       fetch: accessInfo.fetch,
+      getLogoutUrl: maybeBuildRpInitiatedLogout({
+        idTokenHint: accessInfo.idToken,
+        endSessionEndpoint:
+          oidcLoginOptions.issuerConfiguration.endSessionEndpoint,
+      }),
```

Comment:
This ensures the ID token is bound to the logout URL when doing the refresh token flow.